kuma/settings/common.py

```diff
@@ -867,14 +815,19 @@ def JINJA_CONFIG():
 FLAG_REASONS = DEMO_FLAG_REASONS + WIKI_FLAG_REASONS
 
 # bit.ly
-BITLY_API_KEY = "SET ME IN SETTINGS_LOCAL"
-BITLY_USERNAME = "SET ME IN SETTINGS_LOCAL"
+BITLY_USERNAME = config('BITLY_USERNAME', default='lmorchard')
+BITLY_API_KEY = config('BITLY_API_KEY',
+                       default='R_2653e6351e31d02988b3da31dac6e2c0')
 
-GOOGLE_MAPS_API_KEY = "ABQIAAAAijZqBZcz-rowoXZC1tt9iRT5rHVQFKUGOHoyfP_4KyrflbHKcRTt9kQJVST5oKMRj8vKTQS2b7oNjQ"
+GOOGLE_MAPS_API_KEY = config('GOOGLE_MAPS_API_KEY',
```

Comment:
IIRC, this was only ever used on the events page, which is now removed? We could probably drop this.